spec/features/accessibility/account_pages_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+require 'axe-rspec'
+
+RSpec.feature 'Accessibility on account pages', :js do
+  let(:user) do
+    # Create an interesting user with various states that affect content shown on account pages
+    create(
+      :user,
+      :with_multiple_emails,
+      :with_webauthn,
+      :with_webauthn_platform,
+      :with_phone,
+      :with_piv_or_cac,
+      :with_personal_key,
+      :with_backup_code,
+      :with_authentication_app,
+      :proofed,
```

Comment:
there are so many, WDYT about alphabetizing them? maybe `:proofed` and `:with_mutiple_emails` could go together since are a different category than all the MFAs?

```suggestion
      :proofed,
      :with_multiple_emails,
      # all mfas
      :with_authentication_app,
      :with_backup_code,
      :with_personal_key,
      :with_phone,
      :with_piv_or_cac,
      :with_webauthn,
      :with_webauthn_platform,
```

Reply:
> there are so many, WDYT about alphabetizing them? maybe `:proofed` and `:with_mutiple_emails` could go together since are a different category than all the MFAs?

Makes sense!